src/app/components/header/header.component.ts

```diff
@@ -21,6 +23,19 @@ export class HeaderComponent implements OnInit {
             items: [{ label: 'Project' }, { label: 'Other' }]
           },
           { label: 'Open', routerLink: ['pagename'] },
+          {
```

Comment:
Can we add the "Login" and "Logged In User's name" in the toolbar instead on right aligned menu item?

This way it will be far more "real life"..